### PR TITLE
Us 10226 4 - Apply - Claimant - Welsh Translation for claimant

### PR DESCRIPTION
**Pull request by monalisasarkar123**

### CODE REVIEW

**Review by gautamkumar-hmrc — APPROVED** [review]

**Review by Sasankachyuta13 — APPROVED** [review]

**Review by SushreeM-h — APPROVED** [review]

File: assets/i18n/cy.json [hunk 1]
```diff
@@ -51,11 +51,12 @@
   "MONDAY_TO_FRIDAY_8AM_TO_6PM": "Tu allan i’r DU",
   "ONLINE": "Ar-lein",
   "ASK_HMRC_ONLINE": "Gofyn i CThEF ar-lein",
-  "WE_MAY_CALL_YOU": "We may call you about your claim in case you have a problem.",
+  "WE_MAY_CALL_YOU": "Mae’n bosibl y byddwn yn eich ffonio am eich hawliad os oes gennych broblem.",
   "BEGIN_NEW_CLAIM": "Dechrau hawliad newydd",
   "APPLICATION_FOR_CHB_COMPLETE": "Application for Child Benefit complete",
 
   "APPLICATION_RECEIVED": "Cais am Fudd-dal Plant wedi dod i law",
+  "CLOSE_CLAIM": "Cau’r hawliad",
```
Comment (AkhilaHMRC) [inline assets/i18n/cy.json:59]: this looks to be removed

File: assets/i18n/cy.json [hunk 2]
```diff
@@ -74,7 +75,7 @@
   "RETURN_TO_HOMEPAGE": "Yn ôl i hafan eich porth hawlio",
   "WHAT_DID_YOU_THINK_OF_THIS_SERVICE": "Beth oedd eich barn am y gwasanaeth hwn?",
   "TAKES_30_SECONDS": "(mae’n cymryd 30 eiliad)",
-  "PRINT_THIS_INFO": "Print this page or make a note of your reference number. You will need it if you contact HMRC or HMRC ask you for more information.",
+  "PRINT_THIS_INFO": "Argraffwch y dudalen hon neu gwnewch nodyn o’r cyfeirnod. Bydd angen hwn arnoch os byddwch yn cysylltu â CThEF neu os bydd CThEF yn gofyn am ragor o wybodaeth",
```
Comment (AkhilaHMRC) [inline assets/i18n/cy.json:78]: should there be dot at the end

File: assets/i18n/cy.json [hunk 3]
```diff
@@ -280,21 +281,21 @@
...
+  "DELETE_YOUR_CLAIM": "Dileu eich hawliad",
   "FOR_YOUR_SECURITY_WE_DELETED_YOUR_CLAIM": "For your security, we deleted your claim",
-  "YOU_DELETED_YOUR_CLAIM": "You deleted your claim",
-  "START_CLAIM_AGAIN": "Start claim again",
-  "BEFORE_YOU_GO": "Before you go",
-  "YOUR_FEEDBACK_HELPS_US_MAKES_OUR_SERVICE_BETTER": "Your feedback helps us make our service better",
+  "YOU_DELETED_YOUR_CLAIM": "Gwnaethoch ddileu eich hawliad",
+  "START_CLAIM_AGAIN": "Dechrau hawliad eto",
+  "BEFORE_YOU_GO": "Cyn i chi fynd",
+  "YOUR_FEEDBACK_HELPS_US_MAKES_OUR_SERVICE_BETTER": "Mae’ch adborth yn ein helpu i wella ein gwasanaeth.",
```
Comment (AkhilaHMRC) [inline assets/i18n/cy.json:291]: should the dot be removed at the end

File: assets/i18n/cy.json [hunk 4]
```diff
@@ -304,32 +305,32 @@
...
-  "SELECT_YES_IF_YOU_WANT_TO_CONTINUE_WO_SIGN_IN": "Select yes if you want to continue without signing in",
+  "YES": "Iawn",
+  "NO": "Na",
+  "CONTINUE": "Yn eich blaen",
+  "ARE_YOU_SURE_YOU_WANT_TO_CONTINUE_WO_SIGN_IN": "A ydych yn siŵr eich bod am fynd yn eich blaen heb fewngofnodi?",
+  "SELECT_YES_IF_YOU_WANT_TO_CONTINUE_WO_SIGN_IN": "Dewiswch ‘Iawn’ os ydych am fynd yn eich blaen heb fewngofnodi",
   "I_WANT_TO_CONTINUE_WITHOUT_SIGN_IN": "I want to continue without signing in",
   "I_WANT_TO_SIGN_IN": "I want to sign in",
-  "YOU_CAN_CREATE_A_PERSONAL_GOVT": "You can create a personal Government Gateway account if you do not have one",
+  "YOU_CAN_CREATE_A_PERSONAL_GOVT": "Os nad oes gennych gyfrif personol ar gyfer Porth y Llywodraeth, gallwch greu un.",
```
Comment (AkhilaHMRC) [inline assets/i18n/cy.json:315]: Should there be dot at the end

**Review by AkhilaHMRC — CHANGES_REQUESTED** [review]

File: assets/i18n/cy.json (hunk 1 above)
Comment (monalisasarkar123) [reply assets/i18n/cy.json]: done

File: assets/i18n/cy.json (hunk 3 above)
Comment (monalisasarkar123) [reply assets/i18n/cy.json:290]: done

**Pull request merged**